zebrad/tests/common/lightwalletd/send_transaction_test.rs

```diff
@@ -148,9 +147,6 @@ pub async fn run() -> Result<()> {
 
     let mut rpc_client = connect_to_lightwalletd(lightwalletd_rpc_port).await?;
 
-    // To avoid filling the mempool queue, limit the transactions to be sent to the RPC and mempool queue limits
-    transactions.truncate(max_sent_transactions());
```

Comment:
Note: Removing redundant code like this is great. But if you remember, please mention it in the PR description, so reviewers don't have to search for the reason for the code change.

Reply:
I knew I was forgetting something.